debian/source/format

```diff
@@ -1 +0,0 @@
-3.0 (quilt)
```

Comment:
I'm sorry, I've still not grokked why this is going away. shouldn't it be `3.0 (native)` (https://www.debian.org/doc/manuals/maint-guide/dother.en.html#sourcef) or something?

Reply:
I'm not sure that's relevant. `debian/source/format` is a standard thing for any debian packaging aiui

Reply:
not for ones without source packages, as far as I can tell?